misc/services/mobile-webui/mobile-webui-frontend/src/containers/activities/manufacturing/RawMaterialIssueStepScreen.jsx

```diff
@@ -1,44 +1,39 @@
-import React, { Component } from 'react';
+import React, { PureComponent } from 'react';
 import PropTypes from 'prop-types';
 import counterpart from 'counterpart';
 
+import { getLocation } from '../../../utils';
 import ButtonWithIndicator from '../../../components/ButtonWithIndicator';
 import * as CompleteStatus from '../../../constants/CompleteStatus';
 import { pushHeaderEntry } from '../../../actions/HeaderActions';
 
-class RawMaterialIssueStepScreen extends Component {
-  onScanHUButtonClick = () => {
+class RawMaterialIssueStepScreen extends PureComponent {
+  componentDidMount() {
     const {
-      stepProps: { huBarcode, qtyToIssue },
       dispatch,
-      onScanButtonClick,
+      stepProps: { locatorName },
     } = this.props;
-    onScanButtonClick();
+    const location = getLocation(this.props);
 
     dispatch(
       pushHeaderEntry({
         location,
         values: [
           {
-            caption: counterpart.translate('general.Barcode'),
-            value: huBarcode,
-          },
-          {
-            caption: counterpart.translate('activities.mfg.issues.qtyToIssue'),
-            value: qtyToIssue,
```

Comment:
why the qtyToIssue header entry was removed?

Reply:
Because here we were adding headers for the next screen, which is the scanner. Now scanner handles showing it's own headers.